src/structure_managers/adaptor_neighbour_list.hh

```diff
@@ -780,6 +790,9 @@ namespace rascal {
      */
     size_t n_ghosts;
 
+    //! number of atoms in the unit cell
+    size_t n_atoms{};
+
     //! counts the number of time the neighbour list has been updated
     size_t n_update{0};
```

Comment:
Is this an artefact? Or do you need that for anything specific in the performance analysis?

Reply:
I don't follow you. it is a useful member variable